XCTestBootstrap/TestManager/FBTestManagerAPIMediator.m

```diff
@@ -301,12 +301,12 @@ - (id)_XCT_recordedEventNames:(NSArray *)arg1 timestamp:(NSNumber *)arg2 duratio
 
 - (id)_XCT_testCase:(NSString *)arg1 method:(NSString *)arg2 didFinishActivity:(XCActivityRecord *)arg3
 {
-  return [self handleUnimplementedXCTRequest:_cmd];
+  return nil;
```

Comment:
I was asking your opinion. If you feel like there's _never_ a time we'd want this log, then it's fine.